watchdog.c

```diff
@@ -90,28 +94,34 @@ int start(lua_State *L)
 	return 0;
 }
 
-int stop(lua_State *L)
+static int
+stop(lua_State *L)
 {
 	timeout = 0;
 
-	fiber_cancel(f_petting);
-	f_petting = NULL;
+	if (f_petting != NULL) {
+        fiber_cancel(f_petting);
+        f_petting = NULL;
+	}
 
-	fiber_join(f_timer);
-	f_timer = NULL;
+	if (f_timer != NULL) {
+        fiber_join(f_timer);
+        f_timer = NULL;
+	}
 
 	return 0;
 }
 
-void watchdog_atexit(void)
+static void
```

Comment:
why static?

Reply:
I see no reasons to keep it in "public". Then it was moved to static.